app/models/webhook_header.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: webhook_headers
+#
+#  id         :integer          not null, primary key
+#  webhook_id :integer
+#  name       :string(255)
+#  value      :string(255)
```

Comment:
>>  Right now we want to focus on notifications about new repositories/tags being created. In the near future we will support about notifications for delete events
>
>I'd hope all registry's notifications referring to some repo in the namespace get forwared. Why filter (see you own comment)?

Right now we don't see the need to forward events triggered from portus (like adding a new user, changing the visibility of the repository,...). They are already tracked by the audit feature, that can be later on opened-up via an API if needed.

I see the webhooks as something useful for integrating Portus/distribution into a workflow. So I would expect these webhooks to be used in conjunction with something like Jenkins.